imdb/parser/http/movieParser.py

```diff
@@ -253,11 +253,11 @@ class DOMHTMLMovieParser(DOMParserBase):
                     path=".//td[starts-with(text(), 'Aspect')]/..//li/text()",
                     postprocess=lambda x: x.strip()
                 ),
-                Attribute(
-                    key="mpaa",
-                    path=".//td/a[starts-with(text(), 'MPAA')]/../../div/text()",
-                    postprocess=lambda x: x.strip()
-                ),
+                # Attribute(
+                #     key="mpaa",
```

Comment:
Is there an alternative to fetching MPAA rating? 
It is useful information.

Reply:
as far as I know, you can find the MPAA rating in the "certifications" key.

For example:
```
>>> from imdb import IMDb
>>> ia = IMDb() 
>>> m = ia.get_movie('0133093')
>>> m.get('certifications')
['Argentina:13', 'Australia:M', 'Belgium:KT/EA', 'Brazil:12', 'Brazil:14::(TV rating)', 'Canada:14A::(Alberta/British Columbia)', 'Canada:PA::(Manitoba)', 'Canada:14::(Nova Scotia)', 'Canada:AA::(Ontario)', 'Canada:13+::(Quebec)', 'Denmark:15', 'Finland:K-16', 'France:Tous publics::(with warning)', 'Germany:16', 'Hong Kong:IIB', 'Hungary:16', 'Iceland:16', 'India:A', 'Ireland:15::(theatrical)', 'Ireland:18::(video)', 'Israel:PG', 'Italy:T', 'Japan:PG-12', 'Malaysia:18SG', 'Mexico:B', 'Netherlands:16', 'Netherlands:12', 'New Zealand:M', 'Norway:15', 'Peru:14', 'Philippines:PG-13', 'Portugal:M/12', 'Russia:16+', 'Singapore:PG', 'Singapore:PG13', 'South Africa:10', 'South Korea:12', 'Spain:18', 'Sweden:15', 'Switzerland:12::(canton of Geneva)', 'Switzerland:12::(canton of Vaud)', 'United Kingdom:15', 'United States:R']
```

You can just iterate over the list and find the entry that starts with "United States:".